model-optimizer/extensions/front/tf/pooling_ext.py

```diff
@@ -72,20 +72,44 @@ def extract(cls, node):
         return cls.enabled
 
 
-def create_pooling_attrs(node, pool_method):
+class AvgPoolV2FrontExtractor(FrontExtractorOp):
+    op = 'AvgPoolV2'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        attrs = create_pooling_attrs(node, 'avg')
```

Comment:
I don't see initialization of `is_pooling_v2` argument here.

Reply:
Thanks for noting that! Fixed